wordpress-mobile/WordPress-Android
# Add cache to Insights

Related FluxC PR - https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1156

This PR adds cache to the Insights screen. The changes are mostly FluxC but there are some details in here too. We no longer use the data in the `OnStatsFetched` event but we load them from DB instead. The load from DB in the `ViewAll` case in Followers loads now everything in the DB (so we can load the cached pages).

To test:
* Go to Stats/Insights
* Try leaving and reentering the screen
* Go to View All pages
* Go to Followers/View All and leave
* The app should only load data once
* The app should refresh data on PTR
* Use Stetho to check if the app makes multiple requests
